src/Codeception/Test/Loader/Gherkin.php

```diff
@@ -140,6 +140,7 @@ protected function addSteps(array $contexts, string $group = 'default'): void
                 $annotation = Annotation::forMethod($context, $method);
                 foreach (['Given', 'When', 'Then'] as $type) {
                     $patterns = $annotation->fetchAll($type);
+                    codecept_debug($patterns);
```

Comment:
Probably this debug statement is not necessary.

Reply:
ups, sorry :)